js/app.jsx

```diff
@@ -517,6 +537,8 @@ var Job = React.createClass({
       ['State', state],
       ['Input', inputs],
       ['Output', cleanJobPath(job.conf.output)],
+      ['Pipe', pipe],
+      ['Step', step],
```

Comment:
From what I've gathered, a step is a single MR job and a pipe is like a chain of jobs. It might help to call it Scalding Pipes and Scalding Steps? Not too familiar with Scalding to suggest better names. @eigenvariable @alyssafrazee Any thoughts?